File: smoke_tests/javascript/hardcoded_secret.js
// --- VULNERABLE CODE ---
const config = {
  apiKey: "ak_test_a1b2c3d4e5f6g7h8i9j0", // CWE-798: Use of Hard-coded Credentials
  apiSecret: "secret-key-from-a-service-provider", // CWE-798
  password: "MySuperSecurePassword123" // CWE-798
};
// -----------------------

function connectToApi() {
  console.log(`Connecting with key: ${config.apiKey}`);
}

connectToApi();

File: smoke_tests/python/hardcoded_secret.py
# --- VULNERABLE CODE ---
# Hardcoded API key, password, or other secret
API_KEY = "sk_live_abcdefghijklmnopqrstuvwxyz1234567890" # CWE-798: Use of Hard-coded Credentials
PASSWORD = "admin_password_123!" # CWE-798
# -----------------------

def connect_to_service():
    print(f"Connecting with API Key: {API_KEY}")

def login():
    print(f"Logging in with password: {PASSWORD}")

connect_to_service()
login()
